src/Og.php

```diff
@@ -425,6 +431,8 @@ public static function getSelectionHandler(FieldDefinitionInterface $field_defin
    */
   public static function reset() {
     static::$cache = [];
+
+    \Drupal::service('og.access')->reset();
```

Comment:
OgAccessInterface::reset() is deprecated in og:8.1.0-alpha6 and is removed from og:8.1.0-beta1. The static cache has been removed and this no longer server any purpose. See https://github.com/Gizra/og/issues/654

```suggestion
```